src/main/kotlin/racingcar/Race.kt

```diff
@@ -0,0 +1,12 @@
+package racingcar
+
+class Race(numberOfCar: Int) {
+    val carList: List<Car> =
+        List(numberOfCar) { Car(0) }
+
+    fun process() {
+        for (car in carList) {
+            car.process()
+        }
+    }
```

Comment:
```suggestion
    fun process() {
        carList.forEach { 
            it.process()
        }
    }
```

저는 for문보다, forEach문을 사용하여 표현하는 걸 선호합니다. 코드가 더 간결해지고, carList의 it이 car을 의미하기에 더 명확하기 때문이죠.